front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -66,19 +73,8 @@ const StdcmConfig = ({
 
   const { infra } = useInfraStatus();
   const dispatch = useAppDispatch();
-  const { updateStdcmPathStep, restoreStdcmConfig } = useOsrdConfActions() as StdcmConfSliceActions;
-
-  const {
```

Comment:
aren’t we supposed to called the hook the get the right version of the selector?